README.md

```diff
@@ -1 +1,55 @@
-# LeaderBoard
\ No newline at end of file
+# LeaderBoard
+
```

Comment:
- [ ] Please add a short description about your project here to make your readme descriptive :blush: